composer.json

```diff
@@ -1,14 +1,29 @@
 {
     "name": "global-devteam/laravel-api-password-reset",
     "description": "This package implements password reset for APIs",
-    "type": "project",
+    "type": "library",
     "license": "MIT",
     "authors": [
         {
             "name": "Carlos Albuquerque Carneiro",
             "email": "carloscarneiropmw@gmail.com"
         }
     ],
-    "minimum-stability": "stable",
-    "require": {}
+    "minimum-stability": "dev",
+    "require": {
+        "illuminate/support": "^5.0|^6.0"
```

Comment:
Do we need to explicitly state OR 6.0?

Wouldn't it suffice to say "^5" because 6 is greater than 5?

Reply:
@beauraines no we dont. I will remove that